src/fsharp/TcGlobals.fs

```diff
@@ -1058,6 +1058,9 @@ type public TcGlobals(compilingFslib: bool, ilg:ILGlobals, fslibCcu: CcuThunk, d
   member val system_MarshalByRefObject_tcref =  tryFindSysTyconRef sys "MarshalByRefObject"
   member val system_MarshalByRefObject_ty = tryMkSysNonGenericTy sys "MarshalByRefObject"
 
+  member val system_ExceptionDispatchInfo_ty =
```

Comment:
I realize none of the surrounding ones here have a `///` comment, but it would be nice to have it added for the sake of tooling if this needs to get used again elsewhere. These members should all have these kinds of comments, perhaps I can snipe @dsyme into doing that one day 🙂

Reply:
Snipe away!